### PR TITLE
[Notifier] Add Sms77 Bridge

### PR DESCRIPTION
**Pull request by matthiez**

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -653,9 +654,9 @@ all configured texter and chatter transports only in the ``dev`` (and/or
                 slack: 'null://null'
 
 .. TODO
-    - Using the message bus for asynchronous notification
-    - Describe notifier monolog handler
-    - Describe notification_on_failed_messages integration
+- Using the message bus for asynchronous notification
+- Describe notifier monolog handler
+- Describe notification_on_failed_messages integration
```
Comment (OskarStark) [inline notifier.rst:659]: This should be reverted

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Comment by OskarStark** [comment]
@matthiez the code is merged, can you please finish this PR? Thank you

**Comment by OskarStark** [comment]
Thanks for your work on this new feature!

**Comment by OskarStark** [comment]
I fixed it while merging, thanks

File: notifier.rst (hunk 1 above)
Comment (matthiez) [reply notifier.rst:659]: Sorry, seems like I forgot about it. Thanks for your help!

**Pull request closed without merging**